testutil/network/network.go

```diff
@@ -217,24 +289,24 @@ func New(t *testing.T, cfg Config) *Network {
 		appCfg.GRPCWeb.Enable = false
 		if i == 0 {
 			apiListenAddr, _, err := server.FreeTCPAddr()
-			require.NoError(t, err)
+			t.AssertNoError(err)
```

Comment:
Can this `New` function return an error, so that we maybe don't need AssertNoError?

Reply:
Yep, i'll give that a shot. Think this will be cleaner thx.